server/src/main/java/com/lunchtime/service/implementation/OrderServiceImplementation.java

```diff
@@ -0,0 +1,66 @@
+package com.lunchtime.service.implementation;
+
+import com.lunchtime.models.Order;
+import com.lunchtime.models.Person;
+import com.lunchtime.models.RestaurantTable;
+import com.lunchtime.repository.OrderRepository;
+import com.lunchtime.service.OrderService;
+import com.lunchtime.service.PersonService;
+import com.lunchtime.service.RestaurantTableService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class OrderServiceImplementation implements OrderService {
+
+    private final OrderRepository orderRepository;
+    private final RestaurantTableService restaurantTableService;
+    private final PersonService personService;
+
+    public OrderServiceImplementation(
+        OrderRepository orderRepository,
+        RestaurantTableService restaurantTableService,
+        PersonService personService
+    ) {
+        this.orderRepository = orderRepository;
+        this.restaurantTableService = restaurantTableService;
+        this.personService = personService;
+    }
```

Comment:
**[@RequiredArgsConstructor](https://projectlombok.org/features/constructor)** will make code easier (it creates constructor with reqired fields `final` fields in this case).

Reply:
Thanks! Very  useful!